images/openshift-enterprise.yml

```diff
@@ -24,6 +24,7 @@ labels:
   io.k8s.display-name: OpenShift Container Platform Application Platform
   io.openshift.tags: openshift,core
   vendor: Red Hat
+mode: disabled
```

Comment:
openshift-enterprise-node still depends on this image. You may not disable it until that image has been given a new parent.

Reply:
It shouldn’t.  I’ll fix that.